Classes/OldModel/Event.php

```diff
@@ -4606,20 +4606,18 @@ public function createImageForSingleView($maxImageWidth, $maxImageHeight)
                 'height' => $maxImageHeight,
             ],
         ];
-        $imageWithTag = $this->cObj->cObjGetSingle('IMAGE', $imageConfiguration);
```

Comment:
You'll also need to adapt the unit test accordingly to make the build pass.

Reply:
I don't know where I have to adapt the unit test...
I searched in the ''Tests/Unit/OldModel/EventTest.php'...
And how can I test if my configuration works?

Reply:
At the bottom of the "Coversation" tab of this PR, you will find the check results. Under "All checks have failed", you can click on the "Details" link for Travis CI to see which test has failed.